nimib.nimble

```diff
@@ -1,6 +1,6 @@
 # Package
 
-version       = "0.3.4"
+version       = "0.3.5"
 author        = "Pietro Peterlongo"
```

Comment:
btw, just noticed this but you need to add your name here!

Reply:
Oh :o